app/services/error-handler.js

```diff
@@ -0,0 +1,27 @@
+import Service from '@ember/service';
+import { inject as service } from '@ember/service';
+
+export default class errorHandlerService extends Service {
+  @service('fastboot') fastboot;
+
+  handleServerSideError(error) {
+    if (!this.fastboot.isFastBoot) {return}
+
+    if (typeof FastBootRaven !== 'undefined') {
+      const req = fastboot.request;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['fastboot' is not defined.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5011987)